websites/M/marcuscodes.me/dist/metadata.json

```diff
@@ -0,0 +1,19 @@
+{
+  "author": {
+    "name": "geneva",
+    "id": "457805013474082817"
+  },
+  "service": "marcuscodes.me",
+  "description": {
+    "en": "The personal website of geneva/marcus."
+  },
+  "url": "marcuscodes.me",
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/DSs0sfV.png",
```

Comment:
Change this logo to something that is not only circle with red colors.

Reply:
That's the website logo.

Reply:
Should I change it to something else other than the favicon?

Reply:
I mean... yes... a red dot aint really a logo imo